main.go

```diff
@@ -6,32 +6,32 @@ import (
 	"gopkg.in/urfave/cli.v2"
 )
 
-var (
-	version string
-
-	InputFileFlag = &cli.StringFlag{
-		Name:    "input-file",
-		Aliases: []string{"in"},
-		Usage:   "file to convert from",
-	}
-
-	OutputFileFlag = &cli.StringFlag{
-		Name:    "output-file",
-		Aliases: []string{"out"},
-		Usage:   "file to write converted data to",
-	}
-
-	DelimiterFlag = &cli.StringFlag{
-		Name:    "delimiter",
-		Aliases: []string{"d"},
-		Value:   ",",
-		Usage:   "",
-	}
-)
-
 type row map[string]interface{}
 
 func main() {
+	var (
+		version string
```

Comment:
This needs to be global in order to set it in the build command via:
```bash
$ go build -ldflags "-X main.version=v1.0.0"
```

Reply:
agreed on this one!